crates/miden-objects/src/transaction/tx_args.rs

```diff
@@ -96,44 +95,47 @@ impl TransactionArgs {
     // STATE MUTATORS
     // --------------------------------------------------------------------------------------------
 
-    /// Populates the advice inputs with the specified note details.
+    /// Populates the advice inputs with the expected recipient data for creating output notes.
     ///
     /// The advice inputs' map is extended with the following keys:
     ///
-    /// - recipient |-> recipient details (inputs_hash, script_root, serial_num).
+    /// - recipient_digest |-> recipient details (inputs_hash, script_root, serial_num).
     /// - inputs_key |-> inputs, where inputs_key is computed by taking note inputs commitment and
     ///   adding ONE to its most significant element.
```

Comment:
Is this still accurate (i.e., the "adding ONE" part)?

Reply:
Doesn't look like it is, updated it to just `/// - inputs_commitment |-> inputs`